tests/simulator/py/refinement_boxes.py

```diff
@@ -19,7 +19,7 @@ def dup(dic):
         return dic
 
     valid1D = [
-        dup({"refinement_boxes": {"L0": {"B0": [(10,), (50,)]}}}),
+        dup({"refinement_boxes": {"L0": {"B0": [(10,), (14,)]}}}),
```

Comment:
could we instead of 10 and 14 use 10 and nbrGhost-1 or something?
and maybe add a comment that this test case is done so we have only 1 patch on the refined level 1?

Reply:
done